modules/microshift-storage-persistent-storage-reclaim-manual.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage-microshift.adoc
+
+:_content-type: PROCEDURE
+[id="reclaim-manual_{context}"]
+= Reclaiming a persistent volume manually
+
+When a persistent volume claim (PVC) is deleted, the persistent volume (PV) still exists and is considered "released". However, the PV is not yet available for another claim because the data of the previous claimant remains on the volume.
```

Comment:
This is true for cloud storage, but not for the CSI plugin MicroShift ships with.  When a PVC/PV is deleted, the underlying logical volume is handled according to the reclaimPolicy.

Reply:
Updated